utils/keys_test.go

```diff
@@ -298,20 +362,74 @@ func TestImportKeys2InOneFile(t *testing.T) {
 
 	in := bytes.NewBuffer(byt)
 
-	err := ImportKeys(in, []Importer{s})
+	err := ImportKeys(in, []Importer{s}, "", "", passphraseRetriever)
+	require.NoError(t, err)
+
+	bFinal, bRest := pem.Decode(s.data["ankh"])
+	require.Equal(t, b.Bytes, bFinal.Bytes)
+	require.Equal(t, "", bFinal.Headers["path"]) // path header is stripped during import
+
+	b2Final, b2Rest := pem.Decode(bRest)
+	require.Equal(t, b2.Bytes, b2Final.Bytes)
+	require.Equal(t, "", b2Final.Headers["path"]) // path header is stripped during import
+	require.Len(t, b2Rest, 0)
+
+	cFinal, cRest := pem.Decode(s.data["morpork"])
+	require.Equal(t, c.Bytes, cFinal.Bytes)
+	require.Equal(t, "", bFinal.Headers["path"])
+	require.Len(t, cRest, 0)
+}
+
+func TestImportKeys2InOneFileNoPath(t *testing.T) {
+	s := NewTestImportStore()
+
+	b := &pem.Block{
+		Headers: make(map[string]string),
+	}
+	from, _ := os.OpenFile("test.key", os.O_RDONLY, notary.PrivKeyPerms)
+	b.Bytes, _ = ioutil.ReadAll(from)
+	rand.Read(b.Bytes)
+	b.Headers["path"] = "ankh"
+
+	b2 := &pem.Block{
+		Headers: make(map[string]string),
+	}
+	b2.Bytes, _ = ioutil.ReadAll(from)
+	rand.Read(b2.Bytes)
+	b2.Headers["path"] = "ankh"
+
+	c := &pem.Block{
+		Headers: make(map[string]string),
+	}
+	c.Bytes, _ = ioutil.ReadAll(from)
+	rand.Read(c.Bytes)
+	c.Headers["path"] = "morpork"
+
+	bBytes := pem.EncodeToMemory(b)
+	b2Bytes := pem.EncodeToMemory(b2)
+	bBytes = append(bBytes, b2Bytes...)
+	cBytes := pem.EncodeToMemory(c)
+
+	byt := append(bBytes, cBytes...)
+
+	in := bytes.NewBuffer(byt)
+
+	err := ImportKeys(in, []Importer{s}, "", "", passphraseRetriever)
 	require.NoError(t, err)
 
 	bFinal, bRest := pem.Decode(s.data["ankh"])
 	require.Equal(t, b.Bytes, bFinal.Bytes)
-	require.Len(t, bFinal.Headers, 0) // path header is stripped during import
+	require.Equal(t, "", bFinal.Headers["path"]) // path header is stripped during import
```

Comment:
changed this to something like this: 
```
if _, ok := bFinal.Headers["path"]; !ok {
		require.False(t, ok, "expected no path header, should have been removed at import")
	}
```
I don't think NotContains will work since `bFinal.Headers` is a map